### PR TITLE
add protocol and url to loadParameters

### PR DESCRIPTION
**Pull request by dan-barrett**
Closes: https://gitlab.com/architect-io/architect-cli/-/issues/61

### CODE REVIEW

File: src/dependency-manager/src/manager.ts [hunk 1]
```diff
@@ -130,13 +130,31 @@ export default abstract class DependencyManager {
           internal_port = this.toInternalPort(node, interface_name);
         }
 
+        const port = external_host ? external_port : internal_port;
+        const host = external_host ? external_host : internal_host;
+
+        const internal_protocol = 'http';
+        const internal_url = internal_protocol + '://' + internal_host + ':' + internal_port;
+
+        const external_protocol = external_host ?
```
Comment (tjhiggins) [inline src/dependency-manager/src/manager.ts:139]: Can you add an abstract toExternalProtocol method on the manager?
The  host will be localhost when deploying to docker kubernetes and it should be https.

File: src/dependency-manager/src/manager.ts [hunk 2]
```diff
@@ -123,20 +124,39 @@ export default abstract class DependencyManager {
           internal_host = interface_details.host;
           external_port = interface_details.port.toString();
           internal_port = interface_details.port.toString();
+          // external_protocol = interface_details.protocol;
+          // internal_protocol = interface_details.protofol;
+          external_protocol = 'https';
```
Comment (dan-barrett) [inline src/dependency-manager/src/manager.ts:129]: @tjhiggins , @davidthor - do we want to assume `https` for `EXTERNAL_PROTOCOL` of ExternalNodes?

File: src/dependency-manager/src/manager.ts (hunk 1 above)
Comment (dan-barrett) [reply src/dependency-manager/src/manager.ts]: Done. Your comment about not needing changes on the api threw me off of this pattern yesterday.

Comment (tjhiggins) [reply src/dependency-manager/src/manager.ts]: My mistake - just thought of this case

File: src/dependency-manager/src/manager.ts (hunk 2 above)
Comment (tjhiggins) [reply src/dependency-manager/src/manager.ts:129]: Fine for now

Comment (tjhiggins) [reply src/dependency-manager/src/manager.ts:129]: internal_protocol should also be https. then remove comments above

Comment (dan-barrett) [reply src/dependency-manager/src/manager.ts:127]: Done

**Pull request merged**